utils/create-module.js

```diff
@@ -110,22 +100,25 @@ const addImportToSassIndex = (cwd, moduleName) => {
   fs.writeFileSync(indexPath, sortedImports.join('\n'), 'utf8');
 };
 
+const addExportToIndexByType = (cwd, moduleName, moduleType) => {
+  switch (moduleType) {
```

Comment:
This is like deja vu from 30 minutes ago man, I was totally considering a switch but went with an if / else since there's no default case since your throw clause will have already error'd if it's not a valid module type. 6 1/2 dozen one of other though :)